src/libstd/keyword_docs.rs

```diff
@@ -895,6 +895,15 @@ mod mod_keyword {}
 /// // x is no longer available
 /// ```
 ///
+/// `move` is also valid before an async block.
+///
+/// ```rust
+/// let capture = "hello";
+/// let closure = async move || {
```

Comment:
This doc says "async block" above, but uses an async closure and says "async closure" below. async closures are not stable and shouldn't be mentioned in these docs.